Cargo.toml

```diff
@@ -351,21 +351,23 @@ alloy-dyn-abi = "0.7.2"
 alloy-sol-types = "0.7.2"
 alloy-rlp = "0.3.4"
 alloy-trie = "0.4"
-alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-rpc-types-anvil = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-rpc-types-trace = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-rpc-types-beacon = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-genesis = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
-alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d" }
+alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "14ed25d", default-features = false, features = [
+    "eth",
+]}
```

Comment:
I think the idea was to enable this feature in whatever crates requires it, rather than enable it at the top level

Reply:
for this specific crate this makes sense, because this is just a re-export